src/Features/Blockcore.Features.Wallet/Database/WalletStore.cs

```diff
@@ -73,7 +69,7 @@ public WalletStore(Network network, DataFolder dataFolder, Types.Wallet wallet)
                 // Attempt to access the user version, this will crash if the loaded database is V5 and we use V4 packages.
                 try
                 {
-                    var walletVersion = this.sqliteConnection.QueryFirst<int>("SELECT WalletVersion FROM WalletData");
+                    var walletVersion = this.GetDbConnection().QueryFirst<int>("SELECT WalletVersion FROM WalletData");
 
                     if (walletVersion != WalletVersion)
```

Comment:
this also needs a fix no?

Reply:
Is it fixed? It's calling the method to create a new connection instead of reusing the old one.

Reply:
no I mean the version comparison

Reply:
It was simply VS 2019 playing a trick on me while debugging.